### PR TITLE
Rewrite with svelte

### PR DESCRIPTION
**Pull request by fregante**
This is meant to de-spaghettify the codebase
I don't know when/whether I'll be able to finish this, but I'll open a draft PR in the meanwhile.

### CODE REVIEW

**Comment by fregante** [comment]
It works! Except #106 and #107, which are unrelated to this migration

Testing/review welcome

cc @yakov116 @kidonng @notlmn if you use this

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "indent": ["error", 2],
+    "import/no-unassigned-import": "off",
+    "no-multiple-empty-lines": [
+      "error",
+      {
+        "max": 1,
+        "maxBOF": 2,
+        "maxEOF": 1
+      }
+    ],
```
Comment (notlmn) [inline .eslintrc.json:18]: Why modify this rule, instead of sticking to original max-1-empty-line rule?

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "env": {
+    "browser": true,
+    "webextensions": true
+  },
+  "plugins": ["svelte3", "unicorn"],
+  "extends": ["xo/browser", "plugin:unicorn/recommended"],
+  "rules": {
+    "indent": ["error", 2],
```
Comment (notlmn) [inline .eslintrc.json:9]: Use `xo-space/browser` here and remove the indent rule here?

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "env": {
+    "browser": true,
+    "webextensions": true
+  },
+  "plugins": ["svelte3", "unicorn"],
```
Comment (notlmn) [inline .eslintrc.json:6]: `eslint-plugin-unicorn` is not a direct dependency

Comment (fregante) [reply .eslintrc.json:6]: https://github.com/sveltejs/eslint-plugin-svelte3/issues/41#issuecomment-572503966

I commented the code now

File: .eslintrc.json (hunk 1 above)
Comment (fregante) [reply .eslintrc.json:20]: https://github.com/sveltejs/eslint-plugin-svelte3/issues/41#issuecomment-572503966

I commented the code now

**Comment by fregante** [comment]
Thanks for the review! I'm not super familiar with Svelte yet, so I have trouble figuring out how to split each component/logic. I hope someone knows better

**Comment by fregante** [comment]
Since this works, I'll merge it, so anyone else can keep submitting PRs

**Pull request merged**